packages/runner/lib/sdk/sdk.ts

```diff
@@ -117,6 +117,10 @@ export class NangoActionRunner extends NangoActionBase {
         return this.nango.triggerSync(providerConfigKey, [sync], connectionId, fullResync);
     }
 
+    public startSync(providerConfigKey: string, syncs: (string | { name: string; variant: string })[], connectionId?: string): Promise<void> {
+        return this.nango.startSync(providerConfigKey, syncs, connectionId);
```

Comment:
it's not returning void, it's returning response
```suggestion
        await this.nango.startSync(providerConfigKey, syncs, connectionId);
```

Reply:
True, updated, thanks.